Source/Core/Core/Core.cpp

```diff
@@ -839,16 +839,21 @@ void Callback_VideoCopiedToXFB(bool video_update)
 {
   if (video_update)
     s_drawn_frame++;
+}
 
+// Called at field boundaries in `VideoInterface::Update()`
+bool FrameUpdate(void)
```

Comment:
We typically don't put `void` in the parameter list.

Any particular reason why this returns a value?

Reply:
Nope! Leftover from checking something.